install/install.md

```diff
@@ -254,21 +256,21 @@ Use the selector tool below to select your preferred method, packages, and envir
 <br/>
 <div id="pip"></div>
 
-## 3B. Install RAPIDS with pip (Experimental)
-This is an experimental release supporting single GPU usage. cuDF, dask-cuDF, cuML, cuGraph, RMM and RAFT release 22.10 pip packages are now available. The team is excited to get these packages out into the wild and see the RAPIDS community uses them.
+## 3B. Install RAPIDS with pip
+Beginning with the release of 23.04: cuDF, dask-cuDF, cuML, cuGraph, RMM, RAFT, and cuSpatial CUDA 11 pip packages are generally available.
```

Comment:
Is cuspatial definitely going to make it? I haven't even seen a PR yet here.

Reply:
Good catch! I wrote this a while back when we thought it would happen but some other things got in the way. Will pull it.